lib/vcloud/core/vapp.rb

```diff
@@ -171,6 +181,12 @@ def running?
         vapp[:status].to_i == STATUS::RUNNING ? true : false
       end
 
+      def powered_off?
+        raise "Cannot call stopped? on a missing vApp." unless id
```

Comment:
Also, should this be rephrased per [Dan's comment](https://github.com/gds-operations/vcloud-core/pull/175#discussion_r36169301)?